src/wallet/rl_wallet/rl_wallet.py

```diff
@@ -666,17 +665,17 @@ async def rl_generate_signed_aggregation_transaction(self, rl_info, consolidatin
         agg_spend = CoinSolution(consolidating_coin, Program.to([puzzle, solution]))
 
         list_of_coinsolutions.append(agg_spend)
-        # Spend lock
-        puzstring = f"(r (c (q 0x{consolidating_coin.name().hex()}) (q ())))"
-
-        puzzle = Program.to(binutils.assemble(puzstring))
-        solution = Program.to(binutils.assemble("()"))
-
-        ephemeral = CoinSolution(
-            Coin(self.rl_coin_record.coin.name(), puzzle.get_tree_hash(), uint64(0)),
-            Program.to([puzzle, solution]),
-        )
-        list_of_coinsolutions.append(ephemeral)
+        # # Spend lock
```

Comment:
Again, not sure why this is just commented out rather than deleted.

Reply:
fixed